main.go

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package contracts
+
+//Must be a .go file of some sort in the root, otherwise "make test" will fail.
```

Comment:
I didn't need this in mine. Are you able to run make test in https://github.com/lenny-intel/go-mod-registry/tree/InitialImpl?

Reply:
@lenny-intel But you do have a go file in your root -- interface.go -- which is why it works.

Reply:
aHH, right. I was thinking main.go specifically, not any .go file. Works for me.